Rakefile

```diff
@@ -14,17 +14,18 @@ namespace :spec do
 
   namespace :integration do
     targets = ["ubuntu:trusty"]
+    container_name = 'itamae'
```

Comment:
I think that name is more good if unique using `SecureRandom` .

Like `"itamae-#{SecureRandom.hex(4)}"`

What do you think that @pocke ? :blush:

Reply:
I agree to @pocke